mslice/plotting/plot_window/cut_plot.py

```diff
@@ -193,12 +194,16 @@ def toggle_errorbar(self, line_index, line_options):
                 element.set_alpha(1)
 
     def is_icut(self, is_icut):
+        self._canvas.manager.button_pressed_connected(not is_icut)
+        self._canvas.manager.picking_connected(not is_icut)
```

Comment:
Sorry I missed this before - this object already has access to the `plot_figure_manager`, so you can just do `self.manager.button_pressed_connected`. The same is true in `window_closing`.